--- a/data/org.gnome.Eolie.appdata.xml.in
+++ b/data/org.gnome.Eolie.appdata.xml.in
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>org.gnome.Eolie</id>
+  <developer id="org.gnome">
+    <name>The GNOME Project</name>
+  </developer>
   <launchable type="desktop-id">org.gnome.Eolie.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0+</project_license>
